src/JsonSchema/Constraints/Constraint.php

```diff
@@ -61,11 +61,12 @@ public function setUriRetriever(UriRetriever $uriRetriever)
     /**
      * {@inheritDoc}
      */
-    public function addError($path, $message)
+    public function addError($path, $message, $value = null)
     {
         $this->errors[] = array(
             'property' => $path,
-            'message' => $message
+            'message' => $message,
+            'value' => print_r($value, true)
```

Comment:
@doits Given that the property is provided in an error message, would using a tool like [jq](http://stedolan.github.io/jq/) be better for debugging the JSON file data?